v3/src/components/graph/adornments/univariate-measures/normal-curve/normal-curve-adornment-component.tsx

```diff
@@ -115,14 +115,12 @@
       if (!numericAttrId || !dataConfig) return
       const labelSelection = select(labelRef.current)
       const labelCoords = measure.labelCoords
-      const lineHeight = 20
-      const topOffset = lineHeight * (adornmentsStore?.getLabelLinesAboveAdornment(model) ?? 0)
```

Comment:
By not using getLabelLinesAboveAdornment adornment labels start overlapping. E.g. create a dot plot with a normal curve, mean and median. Then turn on "Show Measure Labels." Observe the difference in main compared to this branch.

Reply:
Ah, I misunderstood the intent. I've restored the default vertical label offsets in another commit which centralizes the computation in the `useAdornmentAttributes()` hook to avoid redundancy.